flight/Modules/VtolPathFollower/vtol_follower_control.c

```diff
@@ -201,48 +387,7 @@ int32_t vtol_follower_control_endpoint(const float dT, const float *hold_pos_ned
 int32_t vtol_follower_control_land(const float dT, const float *hold_pos_ned,
```

Comment:
for now this is good restructuring. in the long run this will 99% be reverted for this method since landing really needs it’s own metric of “completed” that will allow us to spool down motors, etc.